docs/sources/administration/metrics.md

```diff
@@ -8,8 +8,95 @@ parent = "admin"
 weight = 8
 +++
 
-# Internal metrics
+# Internal Grafana metrics
 
-Grafana collects some metrics about itself internally. Currently, Grafana supports pushing metrics to Graphite or exposing them to be scraped by Prometheus.
+Grafana collects some metrics about itself internally. Grafana supports pushing metrics to Graphite or exposing them to be scraped by Prometheus.
 
-To emit internal metrics you have to enable the option under the [metrics] section in your [grafana.ini](http://docs.grafana.org/installation/configuration/#enabled-6) config file. If you want to push metrics to Graphite, you must also configure the [metrics.graphite](http://docs.grafana.org/installation/configuration/#metrics-graphite) section.
+For more information about configuration options related to Grafana metrics, refer to [metrics]({{< relref "../installation/configuration/#metrics" >}}) and [metrics.graphite]({{< relref "../installation/configuration/#metrics-graphite" >}}) in [Configuration]({{< relref "../installation/configuration.md" >}}).
+
+## Available metrics
+
+When enabled, Grafana exposes a number of metrics, including:
+
+* Active Grafana instances
+* Number of dashboards, users, and playlists
+* http status codes
+* Requests by routing group
+* Grafana active alerts
+* Prometheus alerts
+* Grafana performance
+
+## View Grafana metrics in Prometheus
+
+These instructions assume you have already added Prometheus as a data source in Grafana.
+
+1. Enable sending metrics to Prometheus. In your configuration file (`grafana.ini` or `custom.ini` depending on your operating system) remove the semicolon to enable the following configuration options: 
```

Comment:
Something that is different for Prometheus compared to other data sources is that Prometheus collects (scrapes) metrics from applications/servers instead of applications/agents sending metrics to the data source. 

That's why you need to tell Prometheus (as you do on line 62) where it can find Grafana. While for Graphite you configure where to push metrics in Grafana.

Reply:
Do I need to make a change for this?

Reply:
```suggestion
1. Enable Prometheus to scrape metrics from Grafana. In your configuration file (`grafana.ini` or `custom.ini` depending on your operating system) remove the semicolon to enable the following configuration options: 
```

Reply:
Sorry. Should have made a suggestion instead :)